README.md

```diff
@@ -406,11 +437,11 @@ endpoint http:Listener travelAgencyEP {
 service<http:Service> travelAgencyService bind travelAgencyEP {    
 ``` 
 
-- Here we have used ``  @kubernetes:Deployment `` to specify the docker image name which will be created as part of building this service. 
+- Here we have used ``  @kubernetes:Deployment `` to specify the Docker image name which will be created as part of building this service.
```

Comment:
Change 'which' to 'that'. Alternately you can add a comma before 'which'. Using 'that' is better considering the context of the sentence.